unison-src/transcripts/errors/unison-hide-all-error.output.md

```diff
@@ -4,7 +4,7 @@ When an expected error is not encountered in a `unison :hide:all:error` block
 then the transcript parser should print the stanza
 and surface a helpful message.
 
-``` unison :hide:all:error
+``` unison :hide:all :error
```

Comment:
Just curious, why does `:error` get a space but `:all` doesn't?

Reply:
Because `:hide:all` is really just one tag. It’s never been parsable as `:hide :all`. I just changed the printer to put spaces between, but didn’t modify the parsers.

Might be worth renaming to `:hide-all` (perhaps allowing `:hide:all` still for backward compat).

Reply:
I just added a commit with this change, I can back it out if you prefer.